qt-restricted-extras/qscintilla6/gen_qsciscintilla.cpp

```diff
@@ -3433,15 +3433,15 @@ class MiqtVirtualQsciScintilla final : public QsciScintilla {
 		struct miqt_string sigval1 = eventType_ms;
 		void* sigval2 = message;
 		qintptr* result_ret = result;
-		intptr_t* sigval3 = (intptr_t*)(result_ret);
```

Comment:
I think ideally these `uintptr_t` and `intptr_t` would remain as-is

Reply:
I saw that on the review too. The methods that were being removed still need to be addressed as well. I'll see if I can get that sorted now.